helper.py

```diff
@@ -76,7 +78,7 @@ def _download_book(url, book_path):
     if not os.path.exists(book_path):
         with requests.get(url, stream=True) as req:
             path = create_path('./tmp')
-            tmp_file = os.path.join(path, '_-_temp_file_-_.bak')
+            tmp_file = os.path.join(path, f'tmp.{secrets.token_hex(8)}.bak')
```

Comment:
Please don't use feature available only in Python 3.6 and above. You have to consider other Python versions as well.

Reply:
I guess using secrets was a lot simpler, I've updated it to random

Reply:
Yes, a lot easier, but having to install `secret` package and tons of other packages that it is dependent on is just not practical for what you can do easily using a few lines of python statement,